src/app/(main)/page.tsx

```diff
@@ -30,7 +31,34 @@ export default function HomePage() {
     fetchQuestions();
   }, []);
 
+  useEffect(() => {
+    divRef.current?.focus();
+  }, [selectedQuestion]);
+
+  const handleKeyDown = (event: KeyboardEvent<HTMLDivElement>) => {
```

Comment:
Are there packages that can be used instead of this code

Reply:
Not sure what you mean. Can you explain bit more?

Reply:
Are there other ways to handle input text? This is not important